presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -169,6 +169,40 @@ public void testInsertWithFailureDoesntLeaveBehindOrphanedTable()
         }
     }
 
+    @Test
+    public void testDecimalPredicatePushdown()
```

Comment:
test surely does not test what it says it tests. It checks if such query does not fail.

@findepi does it make sense to merge it? Or should we drop it?

Reply:
I think this is valuable, but be sure to add a TODO to make testing less brittle.

Reply:
What do you mean with `to make testing less brittle`?

Reply:
I mean to make the test in such a way that they not only check for correctness (as they try to do now), but also that the pushdown actually occurs (which is now assumed, not tested).

Reply:
This is what I also thought. However `less brittle` is very generic, so I thought you meant something more.